Update layout-helpers.css

elloisemae opened the pull request:
Created a figcaption style for FutureSkills css. @meelijane and @AnthonyArtmann are you able to review?

EF

elloisemae committed:
Update layout-helpers.css

meelijane commented:
Screenshot plz!

meelijane commented:
@maeandco can you add an example of this to the component library page?

elloisemae commented:
Before
![screen shot 2019-02-18 at 9 48 15 am](https://user-images.githubusercontent.com/5388025/52920652-2100f300-3363-11e9-9ff4-7f2b7cb2690b.png)

After
![screen shot 2019-02-18 at 9 49 58 am](https://user-images.githubusercontent.com/5388025/52920655-29592e00-3363-11e9-8e32-a7eff64fed80.png)

elloisemae committed:
Update layout-helpers.css

elloisemae commented:
Updated and adjusted the top and bottom margins for some more room. 

After
![screen shot 2019-02-18 at 9 59 13 am](https://user-images.githubusercontent.com/5388025/52920717-de8be600-3363-11e9-9f7e-7c8a9019ca4b.png)

meelijane commented:
@maeandco i'm not sure it looks sufficiently different from the main text. Maybe a light grey background & slightly rounded corners could look good?

meelijane commented on css/layout-helpers.css:
```diff
@@ -5,3 +5,12 @@
 }
+/*Use to caption images in EdX*/
+.figcaption {
+  font-family: Helvetica, Arial, sans-serif;
```
Seeing as `.figcaption` is a class, I assume it will be used on a `<span>` or a `<p>`? Let's add that to your comment above (indication of what HTML element you should use this on). Also, because it will be inheriting styles from `<span>` or `<p>`. we should maybe drop the `font-family` rule because a) I think it's wrong (shouldn't it be Helvetica Neue?) and b) because it should come through from the base element

akflds replied:
I think you talked about this already but just a reminder `<figure>` and `<figcaption>` are elements, not classes :)

elloisemae replied:
Amended ".figcaption" to just "figcaption" after some research, cannot find examples of <p> or <span> used with figcaps.
Corrected the font-family to "Helvetica Neue". As figcap is an element, not sure it will inherit from the base element?

elloisemae replied:
Also
@meelijane, you mentioned that "display:table" wasn't quite right for the "figure" style. Chat when you have a sec? Will research and try to fix myself in the meantime.

meelijane commented on css/layout-helpers.css:
```diff
@@ -5,3 +5,12 @@
+  font-size: 16px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
```
We need to put an example of how to use this in the component library - so people can see what the correct HTML is

elloisemae replied:
Example has been placed in the component library (component.html) in the FutureSkillstheme.

elloisemae committed:
Edit 3

I've left the font-fam info in there for the moment. Adjusted spacing, bg color and restricted width to the image above.

meelijane committed:
Merge branch 'master' into Figcaption-style

elloisemae committed:
Figure and Figcaption changes

Adjusted margins, font-size and background-colour to variables.

elloisemae committed:
Merge branch 'Figcaption-style' of https://github.com/meelijane/futureskillstheme into Figcaption-style

meelijane committed:
Merge branch 'master' into Figcaption-style
